lessons/beginners/list/index.md

```diff
@@ -354,25 +389,34 @@ for cislo in range(10):
 print(mocniny_dvou)
 ```
 
+Podobným způsobem získáš seznam seznam `matka`, `babička`, `prababička`,
+`praprababička`, atd.:
+
+```python
+predkove = ['matka']
+for pocet_pra in range(10):
+    predkove.append('pra' * pocet_pra + 'babička')
+print(predkove)
+```
+
 Chceš-li seznam, který reprezentuje balíček karet,
-zavolej `append` pro všechny kombinace barev a hodnot.
+zavolej `append` pro všechny kombinace barev a hodnot:
 
 ```python
 balicek = []
-for barva in '♠', '♥', '♦', '♣':  # (Na Windows použij textová jména)
+for barva in '♠', '♥', '♦', '♣':
     for hodnota in list(range(2, 11)) + ['J', 'Q', 'K', 'A']:
         balicek.append(str(hodnota) + barva)
```

Comment:
Nějak se mi nechce líbit, že v tom seznamu co procházíme jako _hodnota_ se míchají čísla a řetězce. Nemělo by to převedení na řetězce proběhnout ještě před tím, než začneme sestavovat kombinace?

Reply:
Není to potřeba. A bez *list comprehensions* by se to tím příliš zamotalo.

(Tenhle kód je teď níž, v sekci *Seznamy a náhoda*)